cypress/integration/smokeTests/prodOnly/search.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Remove this file after 1.13 deployed and update package.json "test-staging-no-auth"
+ * and "test-prod-no-auth" to restore the sharedTests version of this file.
+ *
+ * Had to comment out a test because it was triggering the self-DOS
+ * https://ucsc-cgl.atlassian.net/browse/SEAB-3792
+ */
+describe('Admin UI', () => {
+  before(() => {
+    cy.visit('');
+    cy.get('a').contains('Search').click();
+  });
+
+  describe('Basic search functions', () => {
+    it('should arrive on the correct page', () => {
+      cy.url().should('include', '/search');
+    });
+    it('should be able to select facet', () => {
+      cy.get('mat-checkbox').parent().contains('WDL').click();
+      cy.contains('the Language is WDL');
+      cy.url().should('include', '/search?descriptorType=WDL');
+    });
+    it('should be able to handle the back and forward buttons', () => {
+      cy.go('back');
+      cy.url().should('not.include', '/search?descriptorType=WDL');
+      cy.contains('the Language is WDL').should('not.exist');
+      cy.go('forward');
+      cy.url().should('include', '/search?descriptorType=WDL');
+      cy.contains('the Language is WDL').should('exist');
+    });
+    it('should be able to use basic search box and have suggestions', () => {
+      cy.get('[data-cy=basic-search]').type('dockstore_i{enter}');
+      cy.contains(' Sorry, no matches found for dockstore_i');
+      cy.contains(/Do[ ]you[ ]mean:[ ].+/);
```

Comment:
The code robot has a valid beef about these, I think.  It says we should replace the character class with the actual character, for example:
```
cy.contains(/Do you mean: .+/);
```

Reply:
It's not clear from my PR description, but I took "cypress/integration/smokeTests/**sharedTests**/search.ts", copied it to "cypress/integration/smokeTests/**stagingOnly**/search.ts" and "cypress/integration/smokeTests/**prodOnly**/search.ts", then commented out lines 43-55 in the copied versions. Lines 43-55 are what are causing the self-DOSsing.

This is the pattern for running dev/staging/prod specific tests.

Once 1.13 is deployed I will delete this file and the `stagingOnly` copy as well.

TLDR; it's a copy of existing code, and the copy is going to get deleted, so I don't think it's worth it for this PR. But I'll create a followup ticket for the "core" test.

Reply:
On further reflection, I think I'll pass on creating a ticket. The next person who edits the core file can deal with it then, but it's working for now. Feel free to push back.